pom.xml

```diff
@@ -167,6 +167,12 @@
             <version>1.11</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-build-step</artifactId>
```

Comment:
Unnecessary I think; pending [JENKINS-27092](https://issues.jenkins-ci.org/browse/JENKINS-27092), can use non-sandboxed scripts which directly `throw new FlowInterruptedException(…)` for tests.

Reply:
Perhaps this isn't necessary, but I think it makes the test more realistic. I saw that we're already pulling in `pipeline-input-step`, so pulling in `pipeline-build-step` as well didn't seem like a big price to pay for the benefit of more realistic integration testing. I think I'd rather keep this as-is, but let me know if you feel strongly otherwise.

Reply:
No strong feeling.

Reply:
Personally, I'd lean towards the `FlowInterruptedException` approach since it would make `parallelPropagatesStatusImpl` much easier to read, but I think either approach is fine.